### PR TITLE
Test

### PR DESCRIPTION
**Pull request by bossmonkey98**
Cards Component Updated
Apex UI — Mozilla Firefox 12-02-2022 02_07_05
Apex UI — Mozilla Firefox 12-02-2022 02_07_41
Apex UI — Mozilla Firefox 12-02-2022 02_07_48

### CODE REVIEW

File: assets/pages/Cards.css [hunk 1]
```diff
@@ -1,58 +1,139 @@
...
-    grid-template-areas: "img desc";
-    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;}
-#CardImg{
-    display: block;
+    width:80%;
+    max-width: 500px;
+    max-height:400px;
+    display: flex;
+    flex-direction: row;
+    border: 1px solid lightgray;
```
Comment (horsemaker) [inline assets/pages/Cards.css:9]: use a variable instead of `lightgray`

File: assets/pages/Cards.css [hunk 2]
```diff
@@ -1,58 +1,139 @@
...
-    margin: 10px;
-    display: flex;
-    flex-direction: column;
-    width:max-content;
-    grid-area: desc;
+    height: 100%;
+    max-height: 300px;
+    width: 100%;
+    max-width: 200px;
+    border-radius: 5px;
```
Comment (horsemaker) [inline assets/pages/Cards.css:19]: Could we keep a consistent `border-radius`?

File: assets/pages/Cards.css [hunk 3]
```diff
@@ -1,58 +1,139 @@
...
+    justify-content: center;
+    grid-template-columns: 1fr 1fr 1fr;
+    border: 1px solid black;
     border-radius: 10px;
-    overflow: hidden;
-    width: fit-content;
-    height: 400px;
+    background-color:rgba(255, 255, 255, 0.808);
+}
+.vert-card{
```
Comment (horsemaker) [inline assets/pages/Cards.css:55]: Why not `vertical-card`?

File: assets/pages/Cards.css [hunk 4]
```diff
@@ -1,58 +1,139 @@
...
-    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;}
-.footer-content{
-    margin-left:10px ;
+    align-items: center;
+    border: 1px solid var(--secondary);
+    font-size: small;
+    line-height: 18px;
+    letter-spacing: normal;
+    border-radius: 5px;
+    border: 1px solid rgba(0, 0, 0, 0.37);
```
Comment (horsemaker) [inline assets/pages/Cards.css:69]: You could use variables for colors

File: assets/pages/Cards.css [hunk 5]
```diff
@@ -1,58 +1,139 @@
...
+    border-radius: 5px;
+    color:var(--secondary);
+    line-height: normal;
 }
+/* Closwe button */
+
+.close-btn{
+    position: absolute;
+    right: 0px;
+    top: 0px;
```
Comment (horsemaker) [inline assets/pages/Cards.css:122]: Just `0` would work. No need to write `0px`

File: assets/pages/Cards.css [hunk 6]
```diff
@@ -1,58 +1,139 @@
...
+    width: 20px;
+    height: 20px;
```
Comment (horsemaker) [inline assets/pages/Cards.css:124]: Why not rem?

**Review by Omkar-Ghate — COMMENTED** [review]
Good to go after making the mentioned changes.

File: assets/pages/Cards.css [hunk 7]
```diff
@@ -1,58 +1,139 @@
...
+.horizontal-card{
+    background-color: var(--ternary);
     border-radius: 10px;
-    width:max-content;
-    display:grid;
-    border: 1px solid rgba(149, 157, 165, 0.2);
-    grid-template-columns:min-content min-content;
-    grid-template-areas: "img desc";
-    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;}
-#CardImg{
```
Comment (iRohitGaur) [inline assets/pages/Cards.css:9]: any reason for using id here?

File: assets/pages/Cards.css [hunk 8]
```diff
@@ -1,58 +1,139 @@
...
+.vertical-card-container{
+    box-sizing: border-box;
+    padding: 20px;
+    width: 90%;
+    display:grid;
+    column-gap: 10px;
+    align-items: center;
+    justify-content: center;
+    grid-template-columns: 1fr 1fr 1fr;
+    border: 1px solid black;
```
Comment (iRohitGaur) [inline assets/pages/Cards.css:50]: use var

File: assets/pages/Cards.css (hunk 5 above)
Comment (iRohitGaur) [reply assets/pages/Cards.css:122]: +1

File: style.css [hunk 9]
```diff
@@ -36,11 +36,20 @@ i{
...
+    left: 0px;
+    top: 2px;
 }
 
 .name{
     color: var(--primary);
-    margin: 10px;
+    margin-left: 60px;
     font-weight: bolder;
     font-size: 60px;
```
Comment (iRohitGaur) [inline style.css:54]: use rems?

**Pull request merged**